src/tutorial/cli-args.md

```diff
@@ -103,9 +103,11 @@ A much nicer way is to use one of the many available libraries.
 The most popular library for parsing command-line arguments
 is called [`clap`].
 It has all the functionality you'd expect,
-including support for sub-commands, shell completions, and great help messages.
+including support for [sub-commands], [shell completions], and great help messages.
 
 [`clap`]: https://docs.rs/clap/
+[sub-commands]: https://docs.rs/clap/latest/clap/trait.Subcommand.html
```

Comment:
I feel linking to the trait is jumping ahead a bit and this is inconsistent (what do we link to for help?).  I do think shell completions is good as that is a separate crate

Reply:
Make sense, fixed at ea66771e010859e5acbf3691a0fbcc6ef082cdd5